jerry-main/main-unix-snapshot.c

```diff
@@ -357,7 +357,7 @@ process_generate (cli_state_t *cli_state_p, /**< cli state */
       return JERRY_STANDALONE_EXIT_CODE_FAIL;
     }
 
-    FILE *literal_file_p = fopen (save_literals_file_name_p, "w");
+    FILE *literal_file_p = fopen (save_literals_file_name_p, "wb");
```

Comment:
Isn't this a text file?

Reply:
Yes, but I want to preserve the line endings so they are the same on Windows and Linux.